polaris-react/src/components/Pagination/Pagination.scss

```diff
@@ -27,7 +27,7 @@
     border-top: 1px solid var(--p-color-border);
 
     button {
-      --button-min-height: var(--p-space-600);
+      --button-min-height: 28px;
```

Comment:
```suggestion
      --button-min-height: var(--p-height-700);
```

Reply:
nvm it seems like we do have height tokens now, my mistake!

Reply:
Didn't know we have these new tokens, that's great! Will update, thanks folks.